spyder_unittest/widgets/tests/test_unittestgui.py

```diff
@@ -33,7 +31,8 @@ def test_run_tests_and_display_results(qtbot, tmpdir):
     qtbot.wait(1000) # wait for tests to run
```

Comment:
I think this is the reason why tests are failing on qt4.

This PR is already enforcing test passing, so for now all fails :-(